apparmor.d/groups/network/dhcpcd

```diff
@@ -42,13 +42,13 @@ profile dhcpcd @{exec_path} flags=(attach_disconnected) {
   @{PROC}/@{pid}/stat r,
   @{PROC}/sys/kernel/hostname r,
   @{PROC}/sys/net/ipv{4,6}/conf/** r,
-  @{PROC}/sys/net/ipv{4,6}/conf/@{hex}/accept_ra rw,
-  @{PROC}/sys/net/ipv{4,6}/conf/@{hex}/hop_limit w,
-  @{PROC}/sys/net/ipv{4,6}/neigh/@{hex}/{base_reachable_time_ms,retrans_time_ms} w,
```

Comment:
What is the form the the path you have?

Reply:
E.g.:

`/proc/sys/net/ipv6/neigh/enp0s31f6/...`